nike_bot.py

```
@@ -1,63 +1,63 @@
 
-			n = []
+#			n = []
 
-			for sub in s:
-				if 'class="product-card' in sub:
-					try:
-						n.append("{}{}".format(sub, s[s.index(sub) + 1].encode('utf-8')).split("aria-label=")[1].split(" class")[0])
-					except Exception as e:
-						print("{} : {}\n".format(time.time(), e))
-						pass
+#			for sub in s:
+#				if 'class="product-card' in sub:
+#					try:
+#						n.append("{}{}".format(sub, s[s.index(sub) + 1].encode('utf-8')).split("aria-label=")[1].split(" class")[0])
+#					except Exception as e:
+#						print("{} : {}\n".format(time.time(), e))
+#						pass
 
-			x = []
-			st = ""
+#			x = []
+#			st = ""
 
-			for q in n:
-				if "{}\n".format(q) not in o:
+#			for q in n:
+#				if "{}\n".format(q) not in o:
 
-					if len(st) > 1:
-						st = "{}, {}".format(st, q.replace("&#x27;", "'").replace("&amp;", "&").replace("Release Date", ""))
-					else:
-						st = "{}".format(q.replace("&#x27;", "'").replace("&amp;", "&").replace("Release Date", ""))
+#					if len(st) > 1:
+#						st = "{}, {}".format(st, q.replace("&#x27;", "'").replace("&amp;", "&").replace("Release Date", ""))
+#					else:
+#						st = "{}".format(q.replace("&#x27;", "'").replace("&amp;", "&").replace("Release Date", ""))
 
-					x.append(q)
+#					x.append(q)
 
-			st = "".join(i for i in st if ord(i) < 128)
+#			st = "".join(i for i in st if ord(i) < 128)
 
 			# tweet if new products
-			if len(x) > 0:
-				api.PostUpdate(st[:279])
+#			if len(x) > 0:
+#				api.PostUpdate(st[:279])
 
-				with open("/root/dev/product_list.txt", "a") as f:
-					for p in x:
-						f.write("{}\n".format(p))
+#				with open("/root/dev/product_list.txt", "a") as f:
+#					for p in x:
+#						f.write("{}\n".format(p))
 
 			# sleep for one second
-			time.sleep(60.0 - ((time.time() - t) % 60.0))
-		except Exception as e:
-			print("{} : {}\n".format(time.time(), e))
+#			time.sleep(60.0 - ((time.time() - t) % 60.0))
+#		except Exception as e:
+#			print("{} : {}\n".format(time.time(), e))

```

ignore errors that might arise